_targets.R

```diff
@@ -1,38 +1,83 @@
 library(targets)
-source("1_fetch/src/get_nwis_data.R")
+source("1_fetch/src/download_nwis_site_data.R")
+source("1_fetch/src/nwis_site_info.R")
 source("2_process/src/process_and_style.R")
 source("3_visualize/src/plot_timeseries.R")
 
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+# Global parameters
+parameterCd <- '00010'
+startDate <- "2014-05-01" 
+endDate <- "2015-05-01"
+
+
+# For each site, download data and write to csv
+#         Note: this feels inefficient and hard to scale up. How can we "loop" over
+#               multiple sites without making a target specified to each individually?
 p1_targets_list <- list(
   tar_target(
+    site_data_01427207, 
```

Comment:
Since these are saving files, we need to 1) add `format = 'file'` so that targets will treat it as such, 2) make sure that `download_nwis_site_data()` returns the filepath string, and 3) we want to make it clear to a user that the output of this target is a file path by adding the file extension to the target name, e.g. `site_data_01427207_csv`.